scripts/cleanup_submissions/cleanup_submissions.wdl

```diff
@@ -13,21 +15,25 @@ workflow cleanup {
   							  }
   }
 
+  # If the output from task `getSubmissionsInWorkspace` is available, use that list of submissionIds for cleanup.
+  # Else, assume that the user has provided a non-empty array of submissionIds for cleanup.
   Array[String] submissionIdsToClean = select_first([getSubmissionsInWorkspace.submissionIds, submissionIds])
 
-  
-  call getFilesOfSubmissions { input: svcActKeyJson=svcActKeyJson,
-  							      	workspaceProject=workspaceProject,
-  							      	workspaceName=workspaceName,
-  							      	submissionIds=submissionIdsToClean 
-  							}
-  if (!dryRun) {
-  	scatter (fofn in getFilesOfSubmissions.filesToDelete ) {
-  		call httpBatchDelete { input: fofn=fofn, 
-  		                           bucketName=getFilesOfSubmissions.bucketName, 
-  		                           svcActKeyJson=svcActKeyJson 
-  		}
-  	}
+  if (length(submissionIdsToClean) != 0) {
```

Comment:
modified the list/deletes to occur only when the array of submissionIdsToClean isn't empty.